frontend/__tests__/src/services/BoardService.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { setupServer } from 'msw/node'
+import { rest } from 'msw'
+import { boardService } from '@src/services/BoardService'
+import { AxiosError, AxiosResponse } from 'axios'
+import { MOCK_URL } from '../fixtures'
+
+const server = setupServer(
+  rest.get(MOCK_URL, (req, res, ctx) => {
+    return res(ctx.status(200))
+  })
+)
+
+const errorServer = setupServer(
+  rest.get(MOCK_URL, (req, res, ctx) => {
+    return res(
+      ctx.status(403),
+      ctx.json({
+        errorMessage: 'error message',
+      })
+    )
+  })
+)
+
+describe('board service successfully', () => {
+  beforeAll(() => server.listen())
+  afterAll(() => server.close())
```

Comment:
maybe move line 25,26 out of describe, and update fn to beforeEach and afterEach

Reply:
https://mswjs.io/docs/api/setup-server  Refer to msw official website